llvm/lib/CodeGen/MachineLICM.cpp

```diff
@@ -614,6 +608,16 @@ void MachineLICMBase::HoistRegionPostRA(MachineLoop *CurLoop,
       ProcessMI(&MI, RUDefs, RUClobbers, StoredFIs, Candidates, CurLoop);
   }
 
+  // Mark registers as clobbered if they are defined in the loop and also livein
```

Comment:
nit: I would describe this code as ; `if they are livein and also defined in the loop`